pallets/grants/src/lib.rs

```diff
@@ -214,6 +227,91 @@ pub mod pallet {
 
 			Ok(().into())
 		}
+
+		/// Initiate a bridge transfer of all vested funds to the given `eth_address` on the given `chain_id`.
+		/// This process will need to be completed by the bridge oracles.
+		#[pallet::call_index(4)]
+		#[pallet::weight(T::WeightInfo::bridge_all_vesting_schedules())]
+		pub fn bridge_all_vesting_schedules(
+			origin: OriginFor<T>,
+			eth_address: [u8; 20],
+			bridge_id: u32,
+		) -> DispatchResultWithPostInfo {
+			let from = ensure_signed(origin)?;
+			ensure!(
+				Bridges::<T>::contains_key(&BridgeId(bridge_id)),
+				Error::<T>::BridgeNotFound
+			);
+
+			ensure!(!Self::renounced(from.clone()), Error::<T>::Renounced);
```

Comment:
I do not think this is necessary here since we do not really cancel the vesting schedules

Reply:
That's a very good catch. Thank you for spotting it. Copy paste issue

Reply:
I fixed it after adding a test that could catch this issue

Reply:
Vested schedule not canceled -> Tokens claimed -> Token transferred to new account -> token briged ?